services/graphql/src/graphql/definitions/team.js

```diff
@@ -23,6 +23,7 @@ type Team {
   id: String! @projection(localField: "_id")
   name: String! @projection
   description: String @projection
+  active: Boolean!
```

Comment:
If you allow this to be a nullable field, it won't break data if the field is missing